.gitignore

```diff
@@ -105,3 +105,4 @@ ENV/
 
 # Rope project settings
 .ropeproject
+intermediate_source/
```

Comment:
Why is this added to gitignore?

Reply:
I can't remember right now, but it might have been temporary output from the docs build that I wanted to not commit.